.travis.yml

```diff
@@ -0,0 +1,5 @@
+sudo: false
+language: node_js
+node_js:
+  - "4"
+  - "5"
```

Comment:
I mean we used to have a standard CI setup for CL... this is a deviation from that already but I copied what you had for toolbag.